src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -66,6 +82,28 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 		}
 	}
 
+	private _updateSPIDStatus(): void {
+		let activeConnection = TaskUtilities.getCurrentGlobalConnection(this.objectExplorerService, this.connectionManagementService, this.editorService);
+		if (activeConnection) {
+			let uri = this.connectionManagementService.getConnectionUriFromId(activeConnection.id);
+			let info = this.connectionManagementService.getConnectionInfo(uri);
+			if (this.editorService.activeEditor) {
+				// USE ACTIVE EDITOR INFO AS THE SPID WILL BE DIFFERENT FOR EDITOR CONNECTION.
+				let newInfo = this.connectionManagementService.getConnectionInfo(this.editorService.activeEditor.resource.toString());
+				if (newInfo) {
+					info = newInfo;
+				}
+			}
+			let text = '(' + info.spid + ')'
```

Comment:
@erinstellato-ms Thoughts on having the displayed text be more clear? We want to be careful not to have too much long text on the status bar area, but just having parens is pretty easy to miss. 

@cheenamalhotra I wonder if this is something we'd want to put in the tab title as well so users can easily find the query that corresponds to a particular SPID? Maybe something to follow up on, might be worth asking in the original thread to see if people would be interested in that.

Also I noticed that the proposed design you had in that thread is not what we have here - did you decide to go with this route instead?

Reply:
I don't think adding it to tab title would add a huge value as it can change based on when connection refreshes/reconnects.  It would only make it more complex to keep titles/tooltips updated every-time it changes. Showing in status bar seems appropriate to me given we have had this in SSMS in the same way.

> Also I noticed that the proposed design you had in that thread is not what we have here - did you decide to go with this route instead?

The status bar seems fine to me.. though spacing and tooltip text can be slightly adjusted @smartguest ?

Reply:
> If we included additional text, I would argue it needs to be session_id (not SPID), which means it's even more text. Anyone coming from SSMS would be familiar with session_id being in parentheses.
> 
> While session_id is in the tab title in SSMS, it's at the end and everything else, so we'd have to decide whether to be consistent or not if we did that in ADS. I think it's worth asking on the original issue (there is a second one, which also asks for login, where we could ask for opinions #16037) if folks want it in the tab name.

That's what I was thinking, at the end after any truncated content if the name is long. e.g. 

`MyLongQuery.sql (additional login info)...(52)`

Anyways this is a good start so clearly out of the scope of this PR - but if we're working on this stuff it's a good opportunity to try and think how we could improve things overall (even if it's not something SSMS has - we don't need to always copy SSMS exactly)

Reply:
We discussed offline, and will look into adding SPID to editor title in the next release milestone since it would require additional work than we originally planned for. In this PR's scope, we will be adding it to status bar.